redhat/cassandra.spec

```diff
@@ -120,61 +117,47 @@ exit 0
 %files
 %defattr(0644,root,root,0755)
 %doc CHANGES.txt LICENSE.txt README.asc NEWS.txt NOTICE.txt CASSANDRA-14092.txt
-%attr(755,root,root) %{_bindir}/cassandra-stress
-%attr(755,root,root) %{_bindir}/cqlsh
-%attr(755,root,root) %{_bindir}/cqlsh.py
-%attr(755,root,root) %{_bindir}/debug-cql
-%attr(755,root,root) %{_bindir}/nodetool
-%attr(755,root,root) %{_bindir}/sstableloader
-%attr(755,root,root) %{_bindir}/sstablescrub
-%attr(755,root,root) %{_bindir}/sstableupgrade
-%attr(755,root,root) %{_bindir}/sstableutil
-%attr(755,root,root) %{_bindir}/sstableverify
-%attr(755,root,root) %{_bindir}/stop-server
-%attr(755,root,root) %{_sbindir}/cassandra
-%attr(755,root,root) /%{_sysconfdir}/rc.d/init.d/%{username}
-%{_sysconfdir}/default/%{username}
-%{_sysconfdir}/security/limits.d/%{username}.conf
-/usr/share/%{username}*
-%config(noreplace) /%{_sysconfdir}/%{username}
-%attr(755,%{username},%{username}) %config(noreplace) /var/lib/%{username}/*
-%attr(755,%{username},%{username}) /var/log/%{username}*
-%attr(755,%{username},%{username}) /var/run/%{username}*
+%attr(755,root,root) /usr/%{cassandraX}/bin/cassandra-stress
+%attr(755,root,root) /usr/%{cassandraX}/bin/cqlsh
+%attr(755,root,root) /usr/%{cassandraX}/bin/cqlsh.py
+%attr(755,root,root) /usr/%{cassandraX}/bin/debug-cql
+%attr(755,root,root) /usr/%{cassandraX}/bin/nodetool
+%attr(755,root,root) /usr/%{cassandraX}/bin/sstableloader
+%attr(755,root,root) /usr/%{cassandraX}/bin/sstablescrub
+%attr(755,root,root) /usr/%{cassandraX}/bin/sstableupgrade
+%attr(755,root,root) /usr/%{cassandraX}/bin/sstableutil
+%attr(755,root,root) /usr/%{cassandraX}/bin/sstableverify
+%attr(755,root,root) /usr/%{cassandraX}/bin/stop-server
+%attr(755,root,root) /usr/%{cassandraX}/sbin/cassandra
+/usr/share/%{cassandraX}*
+%config(noreplace) /%{_sysconfdir}/%{cassandraX}
+%attr(755,%{username},%{username}) %config(noreplace) /var/lib/%{cassandraX}/*
+%attr(755,%{username},%{username}) /var/log/%{cassandraX}*
+%attr(755,%{username},%{username}) /var/run/%{cassandraX}*
 /usr/lib/python2.7/site-packages/cqlshlib/
 /usr/lib/python2.7/site-packages/cassandra_pylib*.egg-info
 
-%post
-alternatives --install /%{_sysconfdir}/%{username}/conf %{username} /%{_sysconfdir}/%{username}/default.conf/ 0
-exit 0
-
-%preun
-# only delete alternative on removal, not upgrade
-if [ "$1" = "0" ]; then
-    alternatives --remove %{username} /%{_sysconfdir}/%{username}/default.conf/
-fi
-exit 0
-
-
 %package tools
 Summary:       Extra tools for Cassandra. Cassandra is a highly scalable, eventually consistent, distributed, structured key-value store.
 Group:         Development/Libraries
-Requires:      cassandra = %{epoch}:%{version}-%{revision}
+Requires:      cassandra3 = %{epoch}:%{version}-%{revision}
```

Comment:
Shouldn't that be dependent on the matching cassandra version instead of always 3 ?

Reply:
yes, it should) the previous idea was that since it's a `3.xx` branch anyway we do `cassandra3` everywhere, but that's not the best approach. I'll change it.